mne/bem.py

```diff
@@ -1108,12 +1107,112 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
     # Show computed BEM surfaces
     if show:
-        plot_bem(subject=subject, subjects_dir=subjects_dir,
+        plot_bem(subject=subject, sub1jects_dir=subjects_dir,
                  orientation='coronal', slices=None, show=True)
 
     logger.info('Created %s\n\nComplete.' % (fname_head,))
 
 
+def convert_surface(fname_in, fname_out, T1_mgz, useLAS=False):
+    """Convert surface.
```

Comment:
We should say what the conversion is converting to in terms of coordinate systems. It should also help clarify the stuff happening below.